### ISSUE TITLE
Evaluate use of alt text across all images

### ISSUE THREAD

**Issue by edutrul** [bugguide/bugguide#10]
WCAG 2.0 AA standard requires that all non-text content have a text alternative. Images provide this with the alt text attribute which should be a descriptive. The standard: https://www.w3.org/TR/2008/REC-WCAG20-20081211/#text-equiv-all

So to meet this standard, we should make sure all images have alt text, or the opportunity for users to enter it.

For images, the alt attribute meets this standard. The title attribute does not.

For images inside links, the title attribute on either the link or image does not meet this standard. Alt text on the image meets this standard.

Alt text is detected by screen readers and other assistive technology.

**Where images occur: **1. When a user adds an image in an ID request. There is no field for alt text. When the image is published, the main image on the page does not have alt text.2. Image thumbnails. These occur all over. Many have alt text.

Images of this individual.  Alt text appears to be the title of the page.

Browse tab.  Alt text appears to be the title of the page.

Images tab.  Alt text appears to be the title of the page.

3. Images added on Info tabs.  These have an empty alt attribute. VanDyk informs me that they add these images by hand with BBCode. BBC does allow you to add the alt attribute. 

[img alt="my alt text"]imageurl[/img] I suspect this is the same in adding images to comments and forum posts.

I think that if we can accomplish #1 we would be making a major improvement. 

cc  John and  Juampy

References: 
https://isubit.atlassian.net/browse/BUGG-186
https://isubit.atlassian.net/browse/BUGG-187

### Comments:

From Juampy:
Enabling the alt field can be managed via configuration at admin/structure/types/manage/bgimage/fields/field_bgimage_image:

![image](https://user-images.githubusercontent.com/1582129/96152666-68da7380-0ed2-11eb-863b-05353d3a542c.png)

@sacarney , can you try checking that field and then verifying that it works as  you expect it to? If so, can you recreate the bgimage feature? No need to check anything in the features UI, just recreating it should export the changed setting.
